src/components/forms/bouquet/BouquetForm.vue

```diff
@@ -79,8 +79,8 @@ const onValidSubmit = async (validatedValues: {
   // set topic values from validated fields
   topic.value.name = validatedValues.name
   topic.value.description = validatedValues.description
-  topic.value.extras[topicsExtrasKey].theme = validatedValues.theme
-  topic.value.extras[topicsExtrasKey].subtheme = validatedValues.subtheme
+  topic.value.extras![topicsExtrasKey].theme = validatedValues.theme
```

Comment:
Ca a un sens `!` dans le cas d'une assignation ?

Reply:
`'topic.value.extras' is possibly 'undefined'` 😭

Reply:
fixed by #551